ml_model/src/test/java/edu/wm/sealab/featureextraction/FeatureExtractorTest.java

```diff
@@ -72,6 +76,11 @@ public void testMethodParameters() {
 
   @Test
   public void testParenthesis() {
-    assertEquals(NUM_OF_PARANTHESIS, featureVisitor.getFeatures().getParenthesis());
+    assertEquals(NUM_OF_PARANTHESIS, features.getParenthesis());
+  }
+    
+  @Test
+  public void testComments() {
+    assertEquals(NUM_OF_COMMENTS, featureVisitor.getFeatures().getNumOfComments());
   }
 }
```

Comment:
Add more tests to verify avgNumberComments as well

Reply:
Should I be using SCC to count the number of lines of code? If so, how would I do that?
If not, I assume I just manually count the lines, but at that point, it would only really test my ability to do basic math instead of testing how Parser.java acquires the lines of code value in loc_data.csv via SCC.